iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -610,9 +613,7 @@ Inspector.prototype._selectionSummaryAccordian = function() {
                         $("<span></span>") 
                             .addClass("analyse")
                             .text("")
-                            .on("click", () => {
-                                this._chart.analyseDimension(fact, [ aspect.name() ])
-                            })
+                            .on("click", () => this.analyseDimension(fact,[a]))
```

Comment:
I don't understand this change.  The only `a` in scope is `Accordian`?

Reply:
This was a result of botched conflict resolution which accidentally reverted d6e0b256cb9579c904248aedfaa9f54422278fe9 - now fixed